content/applications/finance/accounting/receivables/customer_invoices/cash_discounts.rst

```diff
@@ -1,68 +1,180 @@
-====================
-Offer cash discounts
-====================
-
-Cash discounts are incentives you can offer to customers to motivate
-them to pay within a specific time frame. For instance, you offer a 2%
-discount if the customer pays you within the first 5 days of the
-invoice, when it is due in 30 days. This approach can greatly improve
-your average collection period.
-
-Set up a cash discount
-======================
-
-To set up a cash discount, go to :menuselection:`Accounting -->
-Configuration --> Management --> Payment Terms` and click on
-*Create*. Add a *Percent* type of term with a corresponding value
-(e.g. 98% of the total price for a 2% discount) and the number of days
-during which the offer is valid. You can also change the default balance
-term if needed.
-
-.. image:: cash_discounts/cash_discounts01.png
-  :align: center
-
-Start offering the cash discount
 ================================
+Cash discounts and tax reduction
+================================
+
+**Cash discounts** are reductions in the amount a customer must pay for goods or services offered as
+an incentive for paying their invoice promptly. These discounts are typically a percentage of the
+total invoice amount and are applied if the customer pays within a specified time. The purpose of
+offering cash discounts is to encourage customers to pay their bills quickly, which can help the
+company maintain a steady cash flow.
```

Comment:
```suggestion
total invoice amount and are applied if the customer pays within a specified time.
```
It's already explained at the beginning

Reply:
Thanks, I move the interesting part to the end of the first sentence
", which can help the company maintain a steady cash flow."